### ISSUE TITLE
Errors occuring on the website

### ISSUE THREAD

**Issue by Szymiks** [Code-Poets/project-liberation#82]
I am describing here all errors which occur during testing on different browsers and by different users. There will be available all fixes which users propose and could be implemented.
**Under each photos I describe what bug or incorrect style I found**

# Style
![add_href_to_projects](https://user-images.githubusercontent.com/47628251/68119880-2cea1080-ff04-11e9-8375-d0a2ab3bfeb5.png)
 1. Add links to projects website.

<br>

![estimate_project_should_be_shadow](https://user-images.githubusercontent.com/47628251/68120185-fe206a00-ff04-11e9-9c10-3ebb31a8216a.png)
 1. We can not click button but it should be disable in different style.
EDIT: Will be done in https://github.com/Code-Poets/project-liberation/issues/91
<br>

![footer](https://user-images.githubusercontent.com/47628251/68120187-feb90080-ff04-11e9-8ae8-f39155d8365d.png)
 1. Line in menu should not have space but instead of this it should have white border line in top and bottom.
 2. Search field should have white border around whole div.
 3. If there is not enough conten, footer is not on the bottom
<br>

![img_responsiveness](https://user-images.githubusercontent.com/47628251/68120188-feb90080-ff04-11e9-9529-5a404c5565c3.png)
 1. Image is not responsive
 2. Text in single post should be justify.
<br>

![map_sometimes_work](https://user-images.githubusercontent.com/47628251/68120189-feb90080-ff04-11e9-8541-f29280d289f9.png)
 1. Map sometimes is not rendered.
<br>

![not_align_to_other](https://user-images.githubusercontent.com/47628251/68120191-ff519700-ff04-11e9-988b-87e7dc5d9242.png)
 1. Testimonials doesn't have the same  width as other rows.
 2. About us section should responsive more proportional on big screens , 3000px >>
<br>
 
![same_size_as_other](https://user-images.githubusercontent.com/47628251/68120194-ff519700-ff04-11e9-9f62-a7abbd52bf42.png)
 1. Same as above - testimonial are too width.
<br>

![same_space_on_mobile](https://user-images.githubusercontent.com/47628251/68120195-ffea2d80-ff04-11e9-8bb2-7b8de8ad5ef3.png)
 1. There should be margin top on mobile screens.
<br>

![Screenshot_2019-11-04-11-45-17-134_com android chrome](https://user-images.githubusercontent.com/47628251/68120196-ffea2d80-ff04-11e9-862e-c89e3c5cb1c2.png)
 1. Filtered posts doesn't have margins and some text is cut.

![Screenshot_2019-11-04-11-46-32-823_com android chrome](https://user-images.githubusercontent.com/47628251/68120197-ffea2d80-ff04-11e9-89f5-7038db8d42f9.png)
 1. Margin bottom is to big on mobile screen.
EDIT: Will be done in https://github.com/Code-Poets/project-liberation/issues/103
<br>

![should_not_be_clickable](https://user-images.githubusercontent.com/47628251/68120198-ffea2d80-ff04-11e9-8176-ca28d458d95e.png)
 1. Not displayed divs should not be able to hover.
<br>
 
![smaller_screen](https://user-images.githubusercontent.com/47628251/68120199-0082c400-ff05-11e9-83e3-ffca725d1bbe.png)
 1. Z-index on text div should be bigger than next hotspot z-index and menu z-index.
EDIT: Will be done in https://github.com/Code-Poets/project-liberation/issues/103
<br>

![text_justify](https://user-images.githubusercontent.com/47628251/68120200-0082c400-ff05-11e9-941f-9854f4b576b1.png)
 1. Unexpected error in console on single post view.
<br>

![wrong_display](https://user-images.githubusercontent.com/47628251/68120201-0082c400-ff05-11e9-9988-352d25559f07.png)
1. One of element  has incorrect styles on smaller screen.
EDIT: Will be done in https://github.com/Code-Poets/project-liberation/issues/103

## Linux
<br>
### Firefox

![300px_firefox](https://user-images.githubusercontent.com/47628251/68121635-83f1e480-ff08-11e9-9a94-721b041b9818.png)

 1. On 300px in firefox image is not aligned to center.
<br>
 
### Opera
![opera_delay](https://user-images.githubusercontent.com/47628251/68121704-b69bdd00-ff08-11e9-8c2e-3c0319362c12.png)
 1. Animations are sometimes very delayed, I could reproduce it on opera but I had same problem other browsers, sometimes.
<br>

![quotes](https://user-images.githubusercontent.com/47628251/68122401-8ce3b580-ff0a-11e9-98d1-2a575d7a46a7.png)
 1.  For this particular example quotes are stretched

### Edge

![Edge](https://user-images.githubusercontent.com/47628251/68122581-ee0b8900-ff0a-11e9-98d1-d288f5d7729d.png)

1. Testimonial on Edge is too height same problem occurs on Safari.

<br>

![Edge2](https://user-images.githubusercontent.com/47628251/68122962-eef0ea80-ff0b-11e9-9156-8a9b20b27f9a.png)

 1. Contact us container is not rendered in Edge in proper way.
<br>

![Edge3](https://user-images.githubusercontent.com/47628251/68122989-062fd800-ff0c-11e9-8265-ba506a909780.png)
 1. Menu in Edge is displayed with weird margin.
<br>

### Safari
![IMG_20191104_104153](https://user-images.githubusercontent.com/47628251/68121908-55c0d480-ff09-11e9-9ccf-5f9bf8212a2d.jpg)
 1.  On mobile devices `about us div` is too narrow, but after change window size it is rendered fine.
<br>

![Edge](https://user-images.githubusercontent.com/47628251/68122581-ee0b8900-ff0a-11e9-98d1-d288f5d7729d.png)

1. Testimonial on Safari is too height same problem occurs in edge.

### Edge

![Screenshot_2019-11-04-12-05-53-666_com microsoft emmx](https://user-images.githubusercontent.com/47628251/68122701-3aef5f80-ff0b-11e9-9b68-9dcb7b0f4d85.png)

1. Testimonials are too height
### Edge

![Screenshot_2019-11-04-12-07-06-478_com microsoft emmx](https://user-images.githubusercontent.com/47628251/68122740-52c6e380-ff0b-11e9-8e12-75e4567ea6c4.png)

1.Contact us is too height.
### Opera

![Screenshot_2019-11-04-11-58-01-626_com opera browser](https://user-images.githubusercontent.com/47628251/68122810-7e49ce00-ff0b-11e9-9610-f3f8746b434b.png)

1. Delayed animations.

## IOS
**Not tested**

# User suggestions

## Magda
 - Instagram , facebook etc. should be open in a new tab in the browser.
 - `how we work` section should be easier to found because now it is easy to miss it.

## Sonia
```
- Na głównej grafice brakuje badga z Clutch
- W services dopóki nie będzie landing page’y learn more musi zniknąć, + wychodzi na to, że się zmieni i trzeba będzie dodać cheminformatics / zamienić z czymś innym
- w projects trochę bez sensu, że są klikalne te elementy. w przyszłości jak będą case studies to powinny być i odnosić do stron z nimi
- How we work - nie powinno wyglądać inaczej?
- W teamie przy Adrianie i Kamilu można dodać znaczek linkedina z odnośnikiem do ich profili. Czy w wersji na telefony nie dałoby się skompresować trochę teamu? żeby były np. dwie osoby obok siebie?
- Career - “click here” powinien odnosić tutaj: https://nofluffjobs.com/company/code-poets-ahdxfobg; na telefonie bloczek z "contact us anyway" się rozjechał trochę +
- Blog - nie podoba mi się ta przerwa pomiędzy menu, a paskiem kategorii
```

## Grzesiek
- `estimate project` field in menu should be clickable in whole div
- modal images should be able to exit by escape or click on grayed field
- animations should displayed only once
- checkbox label should be clickable, for example on estimate project template when we would like to click that we agree with privacy and policy it should be clickable on text too.

# If you have any other suggestion feel free to comment.

**Comment by kbeker** [comment]
Most of them are fixed. If any new would occur than we create new issue.
